ts/features/payments/details/store/actions/index.ts

```diff
@@ -35,7 +39,13 @@ export const paymentsTogglePagoPaCapabilityAction = createAsyncAction(
   "PAYMENTS_TOGGLE_PAGOPA_CAPABILITY_CANCEL"
 )<TogglePagoPaCapabilityPayload, void, NetworkError, void>();
 
+/** This action is used to close the PayPal banner into a payment method details of type paypal */
+export const paymentsTogglePayPalBannerAction = createStandardAction(
+  "PAYMENTS_TOGGLE_PAYPAL_BANNER"
+)<void>();
```

Comment:
```suggestion
export const paymentsPayPalBannerVisibilityUpdateAction = createStandardAction(
  "PAYMENTS_TOGGLE_PAYPAL_BANNER"
)<boolean>();
```

What about using an explicit payload (instead of a `toggle` semantic) to handle this action? If for some reason you have an edge case where this action is dipatch twice, you'll have the state set to `true` when you want it to `false`. What do you think about that?

Reply:
Absolutely! The previous implementation was subject to possible bugs and with the suggestion provided by you it is a way stable and robust approach, Thanks! 💪

Addressed into a68e3b698fa39254494ed4a95e81feaa744726d4